cmake-modules/AddTestCMocka.cmake

```diff
@@ -1,39 +1,15 @@
+#
+# Copyright (c) 2007      Daniel Gollub <dgollub@suse.de>
+# Copyright (c) 2007-2018 Andreas Schneider <asn@cryptomilk.org>
+# Copyright (c) 2018      Anderson Toshiyuki Sasaki <ansasaki@redhat.com>
+#
+# Modifed version from https://github.com/xbmc/libssh/blob/master/cmake/Modules/AddCMockaTest.cmake
```

Comment:
Please add more of the comment blocks from this link, including how it is used.
This:
https://github.com/xbmc/libssh/blob/3cb0a1bf023cce7209290f998b8dc3029ed3df77/cmake/Modules/AddCMockaTest.cmake#L6
And this:
https://github.com/xbmc/libssh/blob/3cb0a1bf023cce7209290f998b8dc3029ed3df77/cmake/Modules/AddCMockaTest.cmake#L9-L59

Reply:
Looking at the diff, we should add doc comments about the two new parameters we take, and remove the comment about `LINK_LIBRARIES`: https://www.diffchecker.com/lDkNXNSk

Reply:
done